spec/models/portfolio_spec.rb

```diff
@@ -30,6 +30,18 @@
     end
   end
 
+  context "when updating a portfolio" do
+    let(:workflow_ref) { Time.now.to_i }
+    before do
+      ActsAsTenant.current_tenant = tenant
```

Comment:
@sy Should this be using with_tenant 

https://www.rubydoc.info/gems/acts_as_tenant/0.3.9/ActsAsTenant.with_tenant

Instead of setting the tenant directly.